docs/docsource/Operation-Architecture/Introduction/Conceptual_Framework.rst

```diff
@@ -6,12 +6,12 @@ control, manage, replicate, and visualize data in a multi-cloud context.
 
 Fundamental building blocks of Zenko’s replication framework are the
 bucket, the location, and the cloud service. End users (people or
-services) store data as files in buckets. Administrators are free to
+services) store data as files in buckets. Administrators are free to
 configure and name buckets as they please—to the admin, buckets are
 essentially tenants aggregated in a cloud storage deployment. From an
 end user’s perspective, a bucket simply appears as a networked storage
-location, for example, “accounting,” “bioinformatics-lab,”
-“daily-video-01-07-2019,” or any other organizational unit that makes
+location, for example, “accounting,” “bioinformatics-lab,”
```

Comment:
minor: quotes after the commas

Reply:
Comma inside the quote is a typographical convention, but this is a bit of a corner case/judgment call. If it were a list of parameters that have to be entered verbatim, I would change it for sure. However, this is describing a user's experience, so I'm defaulting to a Chicago running list style.